sdf/kochSDF.glsl

```diff
@@ -0,0 +1,50 @@
+#include "../math/const.glsl"
+
+/*
+original_author: The Art of Code
```

Comment:
maybe we should put the name of the person instead of his youtube channel?

Reply:
I agree.  I used Art of Code because another file referred to the original author that way.